packages/components/src/Breakpoint/Breakpoint.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import React, { useContext, FC, ReactNode, useState } from 'react'
+import { ThemeContext } from 'styled-components'
+import { useResize } from '../utils'
+
+type NamedBreakpoints = 'mobile' | 'tablet' | 'laptop' | 'desktop' | 'xl'
+const NAMED_BREAKPOINTS: NamedBreakpoints[] = [
+  'mobile',
+  'tablet',
+  'laptop',
+  'desktop',
+  'xl',
+]
+export interface BreakpointProps {
+  children: ReactNode
+  from?: NamedBreakpoints
+  to?: NamedBreakpoints
```

Comment:
I think this begs for a tuple.

```suggestion
  display: [ NamedBreakpoints, NamedBreakpoints ]
```

I also think it should be required since the component is kinda silly without this.